### PR TITLE
Add cache for vector chunk

### PR DESCRIPTION
**Pull request by godchen0212**
Add cache for vector chunk
Issue #15540 
Signed-off-by: godchen0212 <qingxiang.chen@zilliz.com>

### CODE REVIEW

**Comment by godchen0212** [comment]
rerun ut

File: internal/querynode/query_collection.go [hunk 1]
```diff
@@ -47,6 +47,10 @@ import (
 	"github.com/milvus-io/milvus/internal/util/typeutil"
 )
 
+var (
+	defaultLocalCacheSize = 64
```
Comment (xiaofan-luan) [inline internal/querynode/query_collection.go:51]: has to be configurable,  and the cache size should be global size of a querynode?

Comment (xiaofan-luan) [reply internal/querynode/query_collection.go:51]: and the cache size should be under MB or GB I guess

**Comment by xiaofan-luan** [comment]
also, we need more stats here in the LRU cache.
Hit ratio, evict ratio, current size and even more....

File: configs/milvus.yaml [hunk 2]
```diff
@@ -143,6 +143,9 @@ queryNode:
   segcore:
     chunkRows: 32768 # The number of vectors in a chunk.
 
+  chunkManager:
+    localFileCacheLimit: 2147483648 # 2 GB, 2 * 1024 *1024 *1024
```
Comment (xiaofan-luan) [inline configs/milvus.yaml:147]: too small, let's set it 16GB by default, and should be under localStorage config

**Pull request merged**